src/shared/sass/_dropdown.scss

```diff
@@ -11,47 +11,79 @@
 	display: block;
 	position: relative;
 	padding-right: 32px;
-}
 
-.button label {
-	pointer-events: none;
-	white-space: nowrap;
-	overflow: hidden;
-	text-overflow: ellipsis;
-	text-transform: none;
-	display: block;
-	width: 100%;
-	font-weight: 400;
-}
+	label {
+		pointer-events: none;
+		white-space: nowrap;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		text-transform: none;
+		display: block;
+		width: 100%;
+		font-weight: 400;
+	}
 
-.button mm-icon {
-	pointer-events: none;
-}
+	#caratIcon {
+		padding: 0 10px 0 13px;
+		position: absolute;
+		margin-top: -4px;
+		top: 50%;
+		right:0px;
+		display: block;
+	}
 
-.button .left-items {
-	@include display(flex);
-	@include align-items(center);
-	@include justify-content(start);
-	height: 100%;
-}
+	#caratIcon.closed {
+		@include transform(scaleY(1));
+	}
 
-#caratIcon {
-	padding: 0 10px 0 13px;
-	position: absolute;
-	margin-top: -4px;
-	top: 50%;
-	right:0px;
-	display: block;
-}
+	#caratIcon.opened {
+		@include transform(scaleY(-1));
+	}
 
-#caratIcon.closed {
-	@include transform(scaleY(1));
-}
+	&.small {
+		padding: 0 26px 0 8px;
+		height: 26px;
+
+		label {
+			font-weight: 700 !important;
```

Comment:
I was thinking that the text was more readable bold @11px... looks pretty fuzzy at the 400 weight, but it could go either way. Your thoughts?